slide-deck.js

```diff
@@ -455,9 +455,12 @@ class slideDeck extends HTMLElement {
     this.goTo(this.slideFromStore());
   }
 
+  // Detect Ctrl / Cmd modifiers in a platform-agnostic way
+  cmdOrCtrl = (event) => event.ctrlKey || event.metaKey;
```

Comment:
We could try to be clever and do platform sniffing to respond to only one of this, but from a quick investigation and issues like [this](https://github.com/vuejs/vue/issues/4843), it seems like just handling both should be enough.